uart_example/README.md

```diff
@@ -0,0 +1,29 @@
+# flutter reactive BLE UART demo app
+
+This demo app shows an usecase for using flutter_reactive_ble library for building a UART emulation over BLE.
```

Comment:
Please, spell "use case" as two separate words and correct the article: "a use case", not ~"an use case"~ ("u" here reads as /ˈjuː/, the first _sound_ is consonant).